shell/platform/windows/public/flutter_windows.h

```diff
@@ -158,6 +158,24 @@ FLUTTER_EXPORT bool FlutterDesktopEngineDestroy(FlutterDesktopEngineRef engine);
 FLUTTER_EXPORT bool FlutterDesktopEngineRun(FlutterDesktopEngineRef engine,
                                             const char* entry_point);
 
+// Creates a view for the given engine.
+//
+// If |engine| is not already running, the view controller will start running
+// it automatically before displaying the window.
```

Comment:
> This sentence is a bit confusing since the document mentions nowhere about displaying the window.

This comment is incorrect: the Windows app runner displays the window after calling this function. This function creates the view controller and launches the engine if needed. FYI, this comment is from the existing API: https://github.com/flutter/engine/blob/ab7d424d4467399d8e62938096237c494488d3d9/shell/platform/windows/public/flutter_windows.h#L79-L80

> Also, I remember we discussed about how creating a view controller needs a HWND. Is it no longer needed?

Similar to how "everything is a widget" in Flutter, in old win32 "everything is an `HWND`". The app has two `HWND`s per window:

1. The window is an `HWND`
2. The Flutter view is backed by an `HWND`

A `FlutterViewController` always needs that second `HWND`, but the first `HWND` is optional.

Reply:
I updated the comment. Let me know if you still have questions or concerns on `HWND`s